.github/workflows/continuous-integration.yml

```diff
@@ -324,7 +324,20 @@ jobs:
       matrix:
         buildtype: [vagovdev, vagovstaging, vagovprod]
 
-    needs: [unit-tests, contract-tests, security-audit, nightwatch]
+    needs: [nightwatch, unit-tests, contract-tests, security-audit, linting]
+
+    # Archive should still run even when linting is skipped (as in PRs).
+    # This if overrides the dependency on linting success to include skipped.
+    # Because of always(), the rest of the needed jobs have to be checked too.
+    # By doing this, later jobs that need this job require similar overrides
+    # or else they will also be skipped when linting is skipped.
+    if: |
```

Comment:
This is not documented at all, but you can use the pipe operator to write a multi-line conditional.

Another undocumented detail is that when using this pipe, you should omit the expression braces (`${{ }}`).

https://github.community/t/how-to-write-multi-line-condition-in-if/128477